marimo/_runtime/runtime.py

```diff
@@ -935,6 +935,8 @@ def mutate_graph(
         LOGGER.debug("Current set of errors: %s", self.errors)
         cells_before_mutation = set(self.graph.cells.keys())
         cells_with_errors_before_mutation = set(self.errors.keys())
+        edges_before = deepcopy(self.graph.children)
+        definitions_before = deepcopy(set(self.graph.definitions.keys()))
```

Comment:
This deepcopy isn't needed because `set` already copies the `DictKeys` datastructure.